Trimble.Modus.Components/Controls/Label/ControlLabel.xaml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             xmlns:local="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
+             x:Class="Trimble.Modus.Components.ControlLabel"
+             x:Name="controlLabelView">
+  <ContentView.Resources>
+    <ResourceDictionary>
+      <toolkit:IsStringNotNullOrEmptyConverter x:Key="IsStringNotNullOrEmptyConverter" />
+      <ResourceDictionary.MergedDictionaries>
+        <local:ControlLabelStyles />
+      </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+  </ContentView.Resources>
+  <Grid Grid.Row="0"
```

Comment:
Any reason to use Grid here? Can't you just the simple StackLayout?

Reply:
Nope. But in some cases stack layout has some UI alignment issues in other projects. so used grid here